source/contribute.rst

```diff
@@ -198,6 +214,31 @@ this in case you want to read more about this:
 - `Commit message style for git <https://commit.style/>`_
 - `A note about git commit messages <https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html>`_
 
+Once you are done, you can push your branch to your fork:
+
+.. code-block:: console
+
+    $ git push --set-upstream fork your-sql-support # use -u for short
+
+Creating the pull request
+-------------------------
+
+New pull requests are created with the repository's default branch as
+base branch, and that might not be what you want. Make sure to pick the
```

Comment:
The base branch should be what you chose according to the guidelines mentioned in the beginning of this chapter. It is also the branch name that you provided when you created your topic branch.